client/state/README.md

```diff
@@ -17,6 +17,26 @@ const store = createReduxStore();
 
 All the application information and data in Calypso should go through this data flow. When you are creating a new module with new data requirements, you should add them to this global store.
 
+You can add a new reducer to the store at runtime easily by wrapping your main component with the `withReducer` Higher Order Component:
+
+```js
+import { withReducer } from 'state/with-reducer';
+
+//...your main React component here...
+
+export default withReducer( 'myModule', myReducerFunction )( MyMainReactComponent );
+```
+
+Alternatively, you can use a more manual approach by adding and removing the reducer yourself. Just make sure you remove the reducer when your code becomes inactive within the system.
+
+```js
+// Call this when your code becomes active.
+store.addReducer( 'myModule', myReducerFunction );
+
+// Call this when your code becomes inactive.
+store.removeReducer( 'myModule' );
```

Comment:
Also, without this code, `replaceReducer` is available, which is far more dangerous than either of these function. Isn't this one step *less* dangerous anyway?